shopify/slack-notification-template/web/routes/index.tsx

```diff
@@ -270,7 +283,13 @@ const ShopPage = () => {
                       displayed by the bot.
                     </Text>
                     <SlackChannelSelectionForm
-                      shop={shop}
+                      shop={
+                        shop as {
```

Comment:
I feel like maybe using type coercion here isn't the way to go
```suggestion
                      shop={{ 
                        id: shop.id,
                        hasSlackAccessToken: shop.hasSlackAccessToken,
                        slackChannelId: shop.slackChannelId 
                      }}
```

Reply:
No need, I've fixed the issue with